changelog.d/8638.bugfix

```diff
@@ -0,0 +1 @@
+User pills don't get lost at message editing anymore
```

Comment:
`pills`, or `polls`?

Reply:
well... i have no idea. i just stole it from [here](https://github.com/vector-im/element-android/pull/8377/files#diff-8856b3acc5e001dcf87438526083a6636bb2ab86740055ddb438b7741c77d3b1) :P maybe the markdown formatting marks are called 'pills' internally?

Reply:
@pludi do you have a better message for the changelog or is anyone working on merging on the PR? the code is not reviewed yet